features.bzl

```diff
@@ -83,6 +83,9 @@ _proto = struct(
 )
 
 _rules = struct(
+    # Whether files with arbitrary characters (including spaces) are allowed in Bazel runfiles.
+    # https://github.com/bazelbuild/bazel/commit/c9115305cb81e7fe645f91ca790642cab136b2a1
+    allow_runfiles_with_arbitrary_characters = ge("7.4.0"),
```

Comment:
I think that `all_characters_allowed_in_runfiles` or similar would sound a bit better in an `if` statement. Open to other suggestions though